tests/kerastuner/engine/base_tuner_test.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2019 The Keras Tuner Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import numpy as np
+
+import kerastuner
+from kerastuner.engine import base_tuner
+
+import tensorflow as tf
+from tensorflow import keras
+
+
+@pytest.fixture(scope='module')
+def tmp_dir(tmpdir_factory):
+    return tmpdir_factory.mktemp('integration_test')
+
+
+def test_base_tuner(tmp_dir):
```

Comment:
Could you also include [a sklearn test](https://github.com/keras-team/keras-tuner/blob/77c59e4567de1bffdbdca4d0271af4f90d3aa5b0/tests/kerastuner/engine/base_tuner_test.py)? It serves as an integration test and solid demonstration that the tuner is abstract and easy to subclass.

Reply:
Done